float32 forward
float32 left
float32 right
float32 rear
float32 shortest_bearing
